packages/communication-react/CHANGELOG.md

```diff
@@ -11,7 +11,19 @@ Fri, 09 Jul 2021 20:41:33 GMT
 
 ### Changes
 
-- Bump react-components to v1.0.0-beta.2 ([PR #496](https://github.com/azure/communication-ui-library/pull/496) by jinan@microsoft.com)
+* Updated calling sdk dependency to 1.2.0-beta.1
+
+* Calling composite improvements including floating local video tile, permission banner when call is being recorded, changing local video device and microphone, and screensharing view improvements
+
+* Chat composite improvements including notifying a participant when they leave a thread and styling improvements.
```

Comment:
nit: consistent period at end of line (some lines end in a period some lines dont)

Reply:
ok lets just remove all of the periods at the end of the lines in patch notes